File: atcoder.jp/contests/abs/tasks/abc086_a/main.cpp
#define _GLIBCXX_DEBUG
#include <bits/stdc++.h>
using namespace std;

template <typename T>
void print(T value) {
  cout << value << endl;
}

bool is_odd(int n) { return n % 2 == 1; }

int main() {
  int A, B;
  cin >> A >> B;

  string ans = is_odd(A * B) ? "Odd" : "Even";
  print(ans);
}

File: atcoder.jp/contests/abs/tasks/abc086_a/main.hs
main :: IO ()
main = do
  [a, b] <- getIntList

  let ans = evenOdd $ a * b
  putStrLn ans

getIntList :: IO [Int]
getIntList = map read . words <$> getLine

evenOdd :: Int -> String
evenOdd x = if odd x then "Odd" else "Even"
